### PR TITLE
Add Emergency Access to Get Premium

### PR DESCRIPTION
**Pull request by vachan-maker**
Added "Emergency Access" to  src/app/settings/premium.component.html and messages.json

### CODE REVIEW

File: src/app/settings/premium.component.html [hunk 1]
```diff
@@ -16,6 +16,9 @@ <h1>{{'goPremium' | i18n}}</h1>
             {{'premiumSignUpTwoStep' | i18n}}
         </li>
         <li>
+            <i class="fa fa-check text-success fa-il" aria-hidden="true"></i>
+            {{'premiumSignUpEmergency' |i18n}}
+        </li>
```
Comment (Hinton) [inline src/app/settings/premium.component.html:21]: We are missing an opening tag here.

Comment (vachan-maker) [reply src/app/settings/premium.component.html:21]: Sorry, but I did not understand what you meant.
I have opened the li tags and closed it as well. Which other tag do I have to include?
Thanks

Comment (cscharf) [reply src/app/settings/premium.component.html:21]: @vachan-maker , you're missing the opening `<li>` tag for the next list item below your change. You have a closing `</li>` tag noted where Oscar added this note, but then if you look below, there's no opening `<li>` tag for the next closing `</li>` tag in the DOM for `{{'premiumSignUpReports' | i18n}}`

File: src/locales/en/messages.json [hunk 2]
```diff
@@ -1565,6 +1565,9 @@
   "premiumSignUpTwoStep": {
     "message": "Additional two-step login options such as YubiKey, FIDO U2F, and Duo."
   },
+  "premiumEmergencyAccess": {
```
Comment (cscharf) [inline src/locales/en/messages.json:1568]: Sorry, just reviewing the rest of these changes now, looks like this key doesn't match what you're referencing in the html component, `premiumSignUpEmergency` vs. `premiumEmergencyAccess`; I can't find the key `premiumSignUpEmergency` anywhere in the current messages.json file.

Comment (vachan-maker) [reply src/locales/en/messages.json]: I have updated it. Sorry for making this complicated.

**Review by cscharf — APPROVED** [review]
No worries, thanks for the contribution!

**Pull request merged**